schema/timestamptz.go

```diff
@@ -90,6 +94,12 @@ func (dst *Timestamptz) Set(src interface{}) error {
 		if originalSrc, ok := underlyingTimeType(src); ok {
 			return dst.Set(originalSrc)
 		}
+		if value, ok := value.(TextMarshaler); ok {
+			s, err := value.MarshalText()
+			if err == nil {
```

Comment:
What happens if `err` is not nil? Do we lose this info?

Reply:
Yeah, we can't really print a warning and using `String()` might just work